eth_tester/backends/pyevm/utils.py

```diff
@@ -27,11 +27,31 @@ def get_pyevm_version():
         return None
 
 
+def get_eels_version():
```

Comment:
This shouldn't belong in pyevm backend. I believe there is a `eels_is_available()` that should help here.